### PR TITLE
OCP4 - au-4 (all) - update to not applicable

### PR DESCRIPTION
**Pull request by itewk**
peer reviewers:
* @curt-matthews
* @mradecker
* @kevindoingbizas
* @thamill

### CODE REVIEW

File: openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml [hunk 1]
```diff
@@ -118,22 +118,32 @@
...
-        A control response is planned. Engineering progress can be tracked via:
+        OpenShift does not control the storage capacity for the
+        OpenShift API audit logs when they are generated.
 
-        https://issues.redhat.com/browse/CMP-159
+        Either the OpenShift API audit logs need to be consumed
+        by an external SEIM and the storage capacity managed there
+        as per organization policy requirments or the operating system on
+        which OpenShift is running needs to have /var/log/audit storage
+        capacity configured to meet organization policy requirments.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml]: Where can documentation be found regarding how to plan appropriate storage capacity for OpenShift?

Also note that, apparently, OpenShift does not audit everything to /var/log/audit/* and has multiple log files on the system. Need to plan for the capacity of those too.

Unfortunately, AU-4(1), which requires off-loading audit records, isn't selected in many baselines. As such we'll need a control response akin to "If not offloading logs, here's how to plan for audit capacity. If offloading logs this control is deferred to external SIEM, and off-loading implementation guidance is provided in AU-4(1)"

File: openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml [hunk 2]
```diff
@@ -118,22 +118,32 @@
...
-        A control response is planned. Engineering progress can be tracked via:
+        OpenShift does not control the storage capacity for the
+        OpenShift API audit logs when they are generated.
 
-        https://issues.redhat.com/browse/CMP-160
+        Either the OpenShift API audit logs need to be consumed
+        by an external SEIM and the storage medium managed there
+        as per organization policy requirments or the operating system on
+        which OpenShift is running needs to have /var/log/audit storage
+        medium configured to meet organization policy requirments.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml]: Note AU-4(1) requires off-loading audit records. Where can documentation be found on how to offload all OCP4 audit records?

Comment (curt-matthews) [reply openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml]: This should be coming from RHCOS and/or RHEL documentation and not OCP documentation.  Therefore, this section of the security controls for those components should have those links.

File: openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml (hunk 1 above)
Comment (shawndwells) [reply openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml]: Further elaboration - when cluster logging is enabled, how do we ensure enough disk space was allocated for Elasticsearch? And on what partition(s) should it be allocated?

The OpenShift docs at https://docs.openshift.com/container-platform/4.3/logging/cluster-logging.html#cluster-logging-about-elasticsearch_cluster-logging redirects to https://www.elastic.co/guide/en/elasticsearch/guide/current/hardware.html, however the Elastic.co documentation is about *hardware*, not sizing.

File: openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml (hunk 2 above)
Comment (shawndwells) [reply openshift-container-platform-4/policies/AU-Audit_and_Accountability/component.yaml]: > This should be coming from RHCOS and/or RHEL documentation and not OCP documentation. Therefore, this section of the security controls for those components should have those links.

OpenShift and CoreOS logging are separate. OpenShift will need its own configuration to forward logs, eg through a ``Log Forwarding Custom Resource``. Much of that now appears documented here:

https://docs.openshift.com/container-platform/4.3/logging/config/cluster-logging-log-forwarding.html

Perhaps some of that language can be used?

**Comment by shawndwells** [comment]
Bump to @itewk. Was the feedback clear and actionable? Will you be able to revisit this PR?

**Comment by itewk** [comment]
Heyo team, can any of you pick this up please:

@curt-matthews
@mradecker
@kevindoingbizas
@thamill
@itewk
@mtclemente
@kpeeples
@dwinchell
@mammerman
@mlcassatt
@siliconjesus
@mertel-rh
@kevin-anderson-redhat

**Comment by redhatrises** [comment]
The comments in this PR haven't been addressed or resolved in a few months. This PR looks frozen. Closing. Please re-open this PR when the comments are addressed appropriately.

**Pull request closed without merging**